sycl/include/sycl/queue.hpp

```diff
@@ -54,14 +54,21 @@
 #define __SYCL_USE_FALLBACK_ASSERT 0
 #endif
 
+// The pi_native_handle2 type has been added as a temporary measure so that the
+// existing pi_native_handle can co-exist with it. At the next ABI redefinition
+// the "2" version will be removed and only pi_native_handle will remain.
+typedef struct _ze_command_queue_handle_t *ze_command_queue_handle_t;
+typedef struct _ze_command_list_handle_t *ze_command_list_handle_t;
+using pi_native_handle2 =
+    std::variant<ze_command_queue_handle_t, ze_command_list_handle_t>;
```

Comment:
Can we move these declarations to [sycl/include/sycl/ext/oneapi/backend/level_zero.hpp](https://github.com/intel/llvm/pull/7196/files#diff-bad45c4e6d46c89d807c929b6adbffdb70993c102647c58b41fb0534e2195727) ?

Reply:
they should just be removed

Reply:
Yes, removed.